package.json

```diff
@@ -4,9 +4,13 @@
   "description": "Google Labs repo for rapid prototyping",
   "type": "module",
   "scripts": {
-    "test": "echo use 'turbo test' instead",
+    "test": "turbo test",
     "build": "turbo build",
-    "clean": "rimraf -g seeds/*/dist seeds/*/.turbo"
+    "lint": "turbo lint",
+    "clean:build": "rimraf -g seeds/*/dist seeds/*/.turbo",
```

Comment:
Could you change `DEVELOPING.md` to reflect this change. I think I have it as `npm run clean` somewhere.